recipes/nextdenovo/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+
+SHARE_DIR=${PREFIX}/share/${PKG_NAME}-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p ${PREFIX}/bin ${SHARE_DIR}
+cp -r ./* ${SHARE_DIR}
+chmod a+x ${SHARE_DIR}/nextDenovo
+ln -s ${SHARE_DIR}/nextDenovo ${PREFIX}/bin/nextDenovo
+export PYTHONPATH=${PYTHONPATH}:${SHARE_DIR}/lib
```

Comment:
This is only run once when the package is built and not when users install the package.

Reply:
ok. Should it be ok with someting like (for the python modules) ?

`ln -s ${SHARE_DIR}/lib/* ${PREFIX}/lib/.`

Reply:
fixed in https://github.com/bioconda/bioconda-recipes/pull/37859/commits/d1e9f33665f83616e483c352e06237e87cb5747f